chart_review/simplify.py

```diff
@@ -38,6 +39,9 @@ def simplify_export(
                 labels |= result_labels
                 text_tags.append(types.LabeledText(result_text, result_labels))
 
+            if grab_all_labels:
+                annotations.labels |= labels
```

Comment:
i had to look |= up - truly we're getting into all the edge case operators with abandon.

Reply:
Hmm, note received. I had thought it was pretty common. Certainly `+=` and `-=` are. But I also like `&=` and this `|=`. I guess I use those more often than the average bear.

Do you prefer this line as: `annotations.labels.update(labels)`?

Reply:
I personally would probably use something closer to that, but it's personal style and i'm fine with what's here, mostly just morning nattering.

Reply:
Love a good morning natter :smile: 

I guess I'll leave it as is, because I do use sets and both `|=` and `&=` a fair bit in this repo. So if we were to do a style switch, it should probably be global. I like the brevity, but that's always a tradeoff..